src/js/main.js

```diff
@@ -203,9 +203,9 @@ function startProcess() {
     }
 
     $('.connect_b a.connect').removeClass('disabled');
-    $('#logo .version, #tab_logoversion .version').text(CONFIGURATOR.version);
-    updateStatusBarVersion();
-    updateTopBarVersion();
+    // with Vue reactive system we don't need to call these,
+    // our view is reactive to model changes
+    // updateTopBarVersion();
```

Comment:
Yep, more for the illustration of Vue 😉 put it so that I didn't forget to comment it.

In general. Will tidy up and split the commits if it comes to merging this.

Reply:
This commentary remains, please remove it. I think is the latest to approve it (at least in my side).